MPSGameLift/Documentation/GameLift.md

```diff
@@ -64,6 +33,7 @@ This README covers optional setup, testing and running on [Amazon GameLift](http
 The "default seed lists" choice should choose all but 4 seed lists to make the engine_pc.pak
 The other 4 seed lists should all get selected to make the game_pc.pak
 It's important to make sure that the bootstrap.game.profile.setreg file has been added to one of the seed files. (also add debug if you want to support debug builds)
```

Comment:
If the location for the setreg file is known, you can validate for this in the export script as well

Reply:
for now, something as simple as a file exists function can help

Reply:
Definitely! I can add checks in the next PR when I actually bundle the assets. Still not sure how we handle that, as a game may have any number of seedlists.